dev/packages/beats/azure/0.0.1/kibana/visualization/b9aa9b20-eaa5-11e9-90ec-112a988266d5.json

```diff
@@ -71,6 +72,7 @@
                         ],
                         "point_size": 0,
                         "separate_axis": 0,
+                        "split_color_mode": "gradient",
```

Comment:
Seems like this change came in in multiple plaecs. Probably a change in KB.

Reply:
Yes, forgot to mention that I refreshed beats.